src/app/pages/instances/components/all-instances/instance-details/instance-devices/add-device-menu/add-device-menu.component.ts

```diff
@@ -104,7 +104,7 @@ export class AddDeviceMenuComponent {
     private loader: LoaderService,
     private snackbar: SnackbarService,
     private translate: TranslateService,
-    private deviceStore: VirtualizationDevicesStore,
+    private deviceStore: VirtualizationVdevsStore,
```

Comment:
This is part of virtualization. Here it is about devices.

Reply:
oh, my mistake!